fxa/tests/test_oauth.py

```diff
@@ -440,6 +440,66 @@ def test_do_not_match_if_provided_is_a_subscope(self):
         self.assertFalse(scope_matches(['abc:xyz'], ['abc']))
         self.assertFalse(scope_matches(['abc:xyz', 'def'], ['abc', 'def']))
 
+    def test_published_test_vectors_for_valid_matches(self):
+        VALID_MATCHES = [
+            ['profile:write', 'profile'],
```

Comment:
do we want to match `[':write', ':write'],` ?

Reply:
or should that throw

Reply:
Do you mean, should we add that as another valid test-case here?  For completeness it should probably throw, but it seems safe to allow it to match in the absence of more thorough validation like that in https://github.com/mozilla/fxa-shared/pull/21

Reply:
> but it seems safe to allow it to match in the absence of more thorough validation 

👍